src/utils.js

```diff
@@ -151,12 +159,42 @@ async function sendMailP(mailOptions) {
 }
 
 function stackToDivs(stack) {
-  return stack?.split('\n').map((line) => `<div>${line}</div>`).join('');
+  return stack
+    ?.split('\n')
+    .map((line) => `<div>${line}</div>`)
+    .join('');
+}
+
+async function checkId(res, next, model, id) {
```

Comment:
can you add this and then I can use the same method when you merge

```
export async function verifySiteId(
  req: express.Request,
  res: express.Response,
  next: NextFunction,
  check: 'param' | 'body' = 'body', // NEW PART
) {
  const { siteId } = check === 'body' ? req.body : req.params;
...
    }
  }
}
```

Reply:
Also I think some other middleware in the lot and maybe job can go away if we use this one

Reply:
Made an issue for this.